api/schemas.yaml

```diff
@@ -658,6 +658,30 @@ ContestSubmissionList:
           outcome_score:
             $ref: "schemas.yaml#/OutcomeScore"
 
+TestCaseList:
+  additionalProperties: false
+  required:
+    - test_cases
+  type: object
+  properties:
+    test_cases:
+      type: array
+      items:
+        type: object
+        additionalProperties: false
+        required:
+          - input
+          - output
+        properties:
+          input:
+            type: string
+            format: byte
+            description: Test input encoded as base64.
+          output:
+            type: string
+            format: byte
+            description: Expected output encoded as base64.
+
```

Comment:
We might either want to specify the order of the example test cases in the `description` or then add a new property specifying the number of the test case.